### ISSUE TITLE
In extended List View I wish to show username only

### ISSUE THREAD

**Issue by tangrit** [jobe-m/ownkeepass#94]
I would be nice, if one could choose to hide the passwords in extended List View but shows the username of the entry.

I have multiple entries for the same machine and encoded the username in the title because I don't want show all the other passwords when searching for a specific entry. When I open an entry, the password is even hidden (correctly :)

**Comment by jobe-m** [comment]
Yes, that should be easily doable. I will take it into consideration for one of the next updates.
